packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -72,7 +72,15 @@ const PipelineRuns = ({
         <StructuredListRow>
           <StructuredListCell>
             {pipelineName ? (
-              <span>No PipelineRuns for {pipelineName}</span>
+              <span>
+                {intl.formatMessage(
+                  {
+                    id: 'dashboard.pipelineRuns.noPipelineRuns',
+                    defaultMessage: 'No PipelineRuns for {pipelineName}'
+                  },
+                  { pipelineName }
+                )}
+              </span>
             ) : (
               <span>No PipelineRuns</span>
```

Comment:
This one should probably be `dashboard.pipelineRuns.noPipelineRuns` and the one above `dashboard.pipelineRuns.noPipelineRunsForPipeline` or similar